engines/stark/services/userinterface.h

```diff
@@ -193,7 +193,7 @@ class UserInterface {
 	void performToggleSubtitle();
 
 	/** Perform an action after a keypress */
-	void handleKeyPress(const Common::KeyState &keyState);
+	void handleKeyPress(const Common::KeyState &keyState, Common::CustomEventType &customType);
```

Comment:
You only really need to pass enum values by reference if you're returning a value. It's more useful for classes and structs to avoid extra copies, while integers can be passed directly in registers.